ingestion/src/metadata/ingestion/source/dashboard/mode/client.py

```diff
@@ -67,15 +67,22 @@ def __init__(self, config):
         )
         self.client = REST(client_config)
 
-    def fetch_all_reports(self, workspace_name: str) -> Optional[list]:
+    def fetch_all_reports(self, workspace_name: str, filter: Optional[str] = None) -> Optional[list]:
```

Comment:
I have added filter as an argument with the default value of None (i.e. no filter). What do you think? @ulixius9

Reply:
Hey @dixon2678 
Adding `all` instead of None as default will bring in all the data
```suggestion
    def fetch_all_reports(self, workspace_name: str, filter: Optional[str] = "all") -> Optional[list]:
```